integrations/pagerduty/pagerduty-incidents/README.md

```diff
@@ -0,0 +1,81 @@
+## Overview
+
+<!-- Sensu Integration description; supports markdown -->
+
+The PagerDuty integration is a Sensu Event Pipeline that generates alerts in the PagerDuty incident response service.
+
+<!-- Provide a high level overview of the integration contents (e.g. checks, filters, mutators, handlers, assets, etc) -->
+
+This integration provides the following resources:
+
+* `pagerduty` [pipeline]
+* `pagerduty` [handler]
+* `pagerduty_integration_key` [secret]
+* `sensu/sensu-pagerduty-handler` [asset]
+
+## Dashboards
+
+<!-- List of supported dashboards w/ screenshots (supports png, jpeg, and gif images; relative paths only; e.g. `![](img/dashboard-1.png)` )-->
+
+This integration is compatible with the [Pagerduty Incidents] dashboard. The Incidents dashboard will display the Sensu Event namespace, entity, and check details (including check status and output).
+
+![](img/dashboard-1.png)
+![](img/dashboard-2.png)
+
+## Setup
+
+<!-- Sensu Integration setup instructions, including Sensu agent configuration and external component configuration -->
+<!-- EXAMPLE: what configuration (if any) is required in a third-party service to enable monitoring? -->
+
+1. **Get a Pagerduty Integration Key**
+
+   To obtain a [Pagerduty Integration Key][pagerduty-integration-key], please visit your Pagerduty dashboard, browse to "Automation" > "Event Rules", and copy your Pagerduty "Integration Key".
+
+1. **Add the Pagerduty Integration Key to your secrets provider**
+
+   Create an environment variable or Hashicorp Vault secret for the Pagerduty Integration Key.
+
+## Plugins
+
+<!-- Links to any Sensu Integration dependencies (i.e. Sensu Plugins) -->
+
+- [sensu/sensu-pagerduty-handler:2.1.0][pagerduty-plugin-bonsai] ([GitHub][pagerduty-plugin-github])
+
+## Metrics & Events
+
+<!-- List of all metrics or events collected by this integration. -->
+
+This integration does not produce any [metrics].
+
+## Alerts
+
+<!-- List of all alerts generated by this integration. -->
+
+This integration does not produce any events that should be processed by an alert or incident management [handler].
```

Comment:
Maybe for the Alerts section in the readme for the pagerduty pipeline, we should say this is the pipeline meant to handle alerts. Instead of the existing boiler plate.  Specifcally mentioning the filters in place by default.

Sure it doesnt create event, but it's meant to process them.

Reply:
Good call. I'll open up a PR to add this since I missed this and already merged. 👍